### ISSUE TITLE
ILL: Error on /cgi-bin/koha/opac-ill.pl

### ISSUE THREAD

**Issue by MagnusEnger** [PTFS-Europe/koha#61]
When I visit /cgi-bin/koha/opac-ill.pl I get this error: 

```
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 79, near "$opt qw( isbn issn title author type start_rec max_results )"
Global symbol "$nav_qry" requires explicit package name at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 99.
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 116, near "}"
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 124, near "}"
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 136, near "}"
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 143, near "}"
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 161, near "}"
syntax error at /home/magnus/scripts/kohaclone/opac/opac-ill.pl line 173, near "}"
Execution of /home/magnus/scripts/kohaclone/opac/opac-ill.pl aborted due to compilation errors.
```

If I change line 79 from this: 

```
for my $opt qw( isbn issn title author type start_rec max_results ) {
```

to this:

```
for my $opt ( qw( isbn issn title author type start_rec max_results ) ) {
```

it works again. 

I think this is a Perl version things, see e.g. http://blogs.perl.org/users/rurban/2010/09/qw-in-list-context-deprecated.html I am running Perl 5.18.2 on Ubuntu 14.04.

**Comment by a-sassmannshausen** [comment]
Ah cheers for the report Magnuse.  I'll look into this!

**Comment by a-sassmannshausen** [comment]
Fixed in 'b416144 Bugfix #61: qw in list context deprecation.'
